README.md

```diff
@@ -1 +1,38 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Brief biography 
+
+My name is **Sruti Kanthan**. I studied psychology in my undergrad, and I am currently a student in the _Masters of Computational Social Science_ program here at UChi. [Here is a link to my github page](https://github.com/skanthan95), where you can see my projects and repositories.
+I was born and raised in New Jersey, have one sister, and a fluffy brown labradoodle named Beau. I moved out to MA after graduating college to work for two years, and then moved to IL for grad school. 
+
+*My hobbies include*:
+- Playing the violin (Celtic or classical)
+- Cooking
+- Trying out fancy food
+- Hiking
+- Singing and listening to music
+
+__My research interests include__:
+  - Sociocognitive development across the lifespan
+  - Communicative competence
+  - Predictors of romantic relationship health
+  
+#### Things I'm doing to stay sane during quarantine:
+1. Watch new shows on Netflix
+    - I highly recommend 'Wilfred'
+2. Do online/virtual museum tours and attend virtual concerts
+3. Just started playing video games from the '80s
+                  
+Here is the image I uploaded as my github avatar, embedded in this README: ![sruti](sruti.jpg). This wasn't working at first because the picture was in the wrong repo, so I moved it and now it should render.
```

Comment:
Yup! It shows up. Nice job embedding an image by citing a local filepath instead of just linking to the static github URL.